Moving FAQs from Hub to GTN (#3308)
galaxyproject/training-material: faqs/galaxy/datasets_finding_datasets.md

---
title: Finding Datasets
area: datasets
box_type: tip
layout: faq
contributors: [jennaj, Melkeb]
---

1. To review all active Datasets in your account, look under User > Datasets.
2. Logging out of Galaxy while the Upload tool is still loading data can cause uploads to abort. This is most likely to occur when a dataset is loaded by browsing local files.

> 3. If you had more than one browser window open, each with a different Galaxy History loaded, the Upload tool will load data into the most recently used history (even if that the view where you originally started the Upload job).

Reviewer: ```suggestion
3. If you have more than one browser window open, each with a different Galaxy History loaded, the Upload tool will load data into the most recently used history.
```

> 4. TIP Refresh the view to display the current active History (Click on refresh icon {% icon galaxy-refresh %} at the top of the History panel). Histories views are cashed by browsers. If your data ended up in an expected History, or if you simply want to organize data, click on gear icon {% icon galaxy-gear %} at the top of the Hisotry panel > "Copy Datasets" function. Exact copies of your own Datasets in multiple Histories do not consume any additional quota space.

Reviewer: The objective of this paragraph is not very clear I think.
